fractal-puppy/Engine
# FEATURE/ Loot Script

Added a new script to spawn prefabs of Items. It can be attached to an Enemy or Chest and once dead or opened it will drop the chosen Items. You only need to tell the name of the prefabs you want to spawn to the script.

This is the first version, there is still work to be done.

**To test:**
1. Open the App and load ItemsTest scene.
2. Hit play and advance until you reach a chest, touch it and see if it drops an Item.
3. Kill the first enemy and see if it drops another item.
4. Check that the items picked work.
